src/main/reactjs/public/i18n/fi-FI.json

```diff
@@ -124,6 +125,48 @@
           "empty": "Tyhjennä valinnat"
         }
       },
+      "clerkTranslatorDetails": {
+        "header": {
+          "personalInformation": "Henkilötiedot",
+          "address": "Osoite",
+          "contactInformation": "Yhteystiedot",
+          "authorisations": "Auktorisoinnit"
+        },
+        "buttons": {
+          "edit": "Muokkaa",
+          "addAuthorisation": "Lisää auktorisointi",
+          "save": "Tallenna",
+          "back": "Takaisin",
+          "next": "Seuraava",
+          "cancel": "Peruuta"
```

Comment:
Näitä ei parhaillaan käytetä, mutta voivat varmaan jäädä odottamaan jatkokehitystä.